specifications/environment-management.org

```diff
@@ -13,83 +13,79 @@ services such as Launchpad.
 Counter that increments every time Snapcraft creates a new build environment.
 Initialized to zero =0=.  Tracked in the environment-manager datastore.
 
-** Project Instance Identification (project-id)
+** Snapcraft Project Path (project-path)
 
-Unique identifier for project, derived from counter =build-count=. The counter
-will be atomically incremented when claiming an identifier.
+Absolute path to a Snapcraft project directory (containing Snapcraft.yaml,
+etc.).
 
-** Build Environment Instance Naming (project-instance-name)
+** Snapcraft Project Name (project-name)
 
-The build environment's user-facing name.
-
-Format: =snapcraft-<project-id>-<project-name>-<project-id>=
-
-** "Stale" Build Environment
+Name of snap (specified in =snapcraft.yaml='s =name= field).
 
-A build environment that is older than 7 days.
+** Snapcraft Project (project-instance)
 
-* Build Instance Lifecycle
+A Snapcraft project, found in =project-path=.
 
-** Creation
+No more than one =project-instance= can exist in =project-path=.
 
-When running =snapcraft= inside a project that requires a build environment,
-Snapcraft shall lookup build environments for user-selected provider and see if
-build-environment already exists and is available for re-use.
+There may be multiple =project-instance(s)= for a given =project-name=
+(typically a user working with multiple project copies/checkouts
```

Comment:
interesting point about checkouts, is the intention to treat a checkout of a branch on a given path as a different project? I suspect not. Maybe just remove "checkouts" if that is the case.